src/index.js

```diff
@@ -0,0 +1 @@
+import './style.css';
```

Comment:
- [REQUIRED] Good job impplementing a nice UI for your app :clap: , however, kindly add render a list of book dynamically from an array as stated in the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=Create%20an%20index.js,values%20for%20each%20task)